.travis.yml

```diff
@@ -55,10 +54,14 @@ script:
   - stty cols $COLUMNS rows 40
   - tput cols; stty size
   - pushd inst
-  - octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "r = octsympy_tests; exit(r)"
+  - if [ "x$PYTAVE" = "xyes" ]; then
+        octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "sympref ipc native; r = octsympy_tests; exit(r)";
+    else
+        octave -W --no-gui --eval "r = octsympy_tests; exit(r)";
+    fi
```

Comment:
Please try to keep the parts of the eval that don't change together. Just set a variable to include an extra few commands in the eval if `$PYTAVE` is enabled.